scripts/parseXML.js

```diff
@@ -1,16 +1,70 @@
 /**
  * Flatten the catalog object
- * @param {array} products 
+ * If objectType is library, there are multiple locations to search in as well as multiple filetypes (pdf, css, html)
+ * We consider other filetypes besides images so that nothing is lost in terms of functionality, at least for the moment.
+ * 
+ * @param {array} sfccObjects
+ * @param {string} objectType
  */
-function getImages(products) {
-    const images = products.filter((product) => product.images)
-    .map((product) => product.images)
-    .reduce((accumulator, arr) => accumulator.concat(arr))
-    .map((group) => group['image-group'])
-    .reduce((accumulator, arr) => accumulator.concat(arr))
-    .map((img) => img.image)
-    .reduce((accumulator, arr) => accumulator.concat(arr))
-    .map(img => img.$.path);
+function getImages(sfccObjects, objectType) {
```

Comment:
could you simplify and create different function for the different usage, it will make clearer to understand

Reply:
Separate functions have been created.